src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age ? <p className="Person__age">{`I am ${person.age}`}</p> : ''}
+    <p className="Person__partner">
+      {person.isMarried
+        ? `${person.partnerName} is my ${person.sex === 'f' ? 'husband' : 'wife'}`
```

Comment:
Nested ternary looks too complicated, let's move this part to separate variable

`const partner = person.sex === 'f' ? 'husband' : 'wife'`